plugin/pulpcore/plugin/changeset/iterator.py

```diff
@@ -32,7 +32,7 @@ class BatchIterator(Iterable):
     """
 
     # Default batch size.
-    BATCH = 1000
+    BATCH = 800
```

Comment:
The BatchIterator is a general purpose object for which the default of 1024 seems reasonable.  The usage specific (more appropriate) size should be passed in.